### ISSUE TITLE
Ctrl+Alt+F[1-7] doesn't work anymore

### ISSUE THREAD

**Issue by infokiller** [DreymaR/BigBagKbdTricks_XKB#2]
When I set the `misc:extend` option in `setxkbmap` Ctrl+Alt+F1 doesn't take me to a virtual console anymore. Do you know what could cause that?

Thanks!

**Comment by DreymaR** [comment]
Yes, I do. I need to make people aware of this in my documentation. The
'XF86_Switch_VT_#' keys that you're used to are implemented in XKB in the
symbols "pc(function)" section (it's in your xkb/symbols directory). You're
likely used to inheriting this via the standard 'pc' settings. My Extend
file specifies new level5–8 mappings for the <FK##> (Function) keys, which
unfortunately for you overwrites this behavior. I think you have to choose
between the server F# keys and my multimedia ones.

Could you try this please, and let me know if it works for you?
In the symbols/extend file under the 'lvl5' section, towards the end of the
section, add a line that includes pc(function), like this (I show some
existing lines to let you know where I mean, only the last line before the
brace is added):
//    include "level5(caps_switch)"
include "pc(function)"
};

If that does the trick, it should be the easiest way to get what you want.
If not, you'll have to delete or comment out the <FK##> lines from the
extend file to get the rest of it without the F# key definitions.

If this works for you, I'll work it into my documentation.

Best of luck,
Øystein "DreymaR"

**Comment by infokiller** [comment]
Thanks!
Adding the line didn't help, but when I disabled the function keys it indeed worked.

**Comment by DreymaR** [comment]
Grrr, I was hoping that re-including the pc(function) definitions after
mine might overwrite the extend ones but no such luck then. Ah well, thanks
for testing it!

**Comment by DreymaR** [comment]
Hey – could you try this then? Replace the <FK01> line in the extend file
with this bit (leaving the 'actions' bit below as it was):
========================================================================
    key <FK01> { symbols[Group1]=[ NoSymbol , NoSymbol , NoSymbol ,
NoSymbol ,
                   XF86AudioPlay        , XF86AudioPause        , //
Play/Pause
                   XF86_Switch_VT_1     , XF86_Switch_VT_1     ], //
Virtual Terminal
//    key <FK01> { symbols[Group1]=[ NoSymbol,NoSymbol,NoSymbol,NoSymbol,
XF86AudioPlay         , XF86AudioPause        ],
========================================================================

What I'm trying to achieve here is making Caps+AltGr(or LCtrl+RAlt if
that's set up as AltGr)+<FK01> to act as the virtual terminal 1 switch.
There might still be a problem with the Level5 CapsState, but please try it.

best,
Øystein

**Comment by infokiller** [comment]
Didn't work, but there's more than a decent chance that I'm doing something wrong- I don't know much about the xkb syntax and it's quite reasonable I made a subtle mistake. BTW, in both cases after doing the change and installing the files setxkbmap gives an uninformative error: `Error loading new keyboard description`.

**Comment by DreymaR** [comment]
Yup, in that case you likely messed up. Easily done! Even in verbose mode,
I don't think you'll get much more info. Guess I'll have to look into it.

**Comment by tecosaur** [comment]
So what's the suggestion here? I'd quite like to leave `Ctrl`+`Alt`+`F1-7` and `Alt`+`SysRq` as is, but I'm not sure what I should do…

**Comment by DreymaR** [comment]
To leave the Ctrl+Alt+F# console functions alone, for now you'll have to forgo those keys' Extend functionality. Find the symbols/extend file and comment out those keys. In a future commit, I may put those in a separate section so you could disable them all using one line.

The SysRq key shouldn't be influenced by my files afaik. If I did include an Extend mapping for it, comment that out likewise.
